internal/controllers/machineset/machineset_controller.go

```diff
@@ -604,34 +641,68 @@ func (r *Reconciler) computeDesiredMachine(machineSet *clusterv1.MachineSet, exi
 	// When we update an existing Machine will we update the fields on the existing Machine (in-place mutate).
 
 	// Set Labels
+	desiredMachine.Labels = machineLabelsFromMachineSet(machineSet)
+
+	// Set Annotations
+	for k, v := range machineSet.Spec.Template.Annotations {
```

Comment:
I know it's trivial at the moment but let's also extract a machineAnnotationsFromMachineSet func to ensure we use the same code in both cases